docs/tasks/tls/managing-tls-in-a-cluster.md

```diff
@@ -195,3 +195,11 @@ noted in the previous section and the reprecussions of issuing a specific
 certificate should be fully understood before granting this permission. See
 [here](/docs/admin/authentication#x509-client-certs) for information on how
 certificates interact with authentication.
+
+## A Note to Cluster Administrators
+
+This tutorial assumes that a signer is setup to serve the certificates API. The
+kubernetes controller manager links a default implementation of a signer. To
```

Comment:
I would either capitalize Kubernetes or refer to it by its binary name, kube-controller-manager.